### DISCUSSION TITLE
What method do I use to check the status of a transaction?

### DISCUSSION THREAD

**Discussion by juankstylez** [TechTailor/Laravel-Binance-Api#6] [General]
I need to pass the ID of the transaction and return the Status, if it is confirmed, pending or canceled! it's possible ? What is the method, I don't see it in the Readme

**Comment by TechTailor** [comment]
Hey,

Sorry, currently the package only supports the methods shown in the readme. I am planning on expanding this package and adding more functionalities soon (probably start working on it little by little from next week onwards), in the meantime, if you can create a PR for the same (if you can), I would be more than happy to check it out.

PS> By transaction status, do you mean for a futures transaction?

-TT
